docs/migrations.rst

```diff
@@ -7,6 +7,27 @@ Migration Guide
 This page contains information about changes between major versions and
 how you can migrate from one version to another.
 
+.. _migration-to-0-13-0:
+
+0.12.x to 0.13.0
+----------------
+
+.. warning::
+
+    This is major new version with a lot of changes under the hood as well
+    as on the API level. Please take a careful look at the mentioned
```

Comment:
hmm, i don't think `with a lot of changes under the hood as well as on the API level` is true 😄 
I think the warning should say we dropped support for py2 and to retrain their models with py3 when switching